### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by luke-den**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+//5)
+int busEffects(bool reverbOn = true, bool delayOn = false, int wetDryMix = 50)
+{
+    ignoreUnused(reverbOn, delayOn, wetDryMix);
+    return {};
+} 
+
+//6)
void defaultLoopLength(int defaultMeasures, int defaultNumOfBeats)
+{
```
Comment (matkatmusic) [inline main.cpp]: Fix this error
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+    float rotations = 2.375f;
+    float exactChange = 12.86f;
+
+    bool doorIsOpen = true;
+    bool makeASound = false;
+    bool activateSwitch = true;
+
+    double numberOfTracks = 4; 
+    double inputsAllowed = 24;
+    double eqModesShown = 6;
```
Comment (matkatmusic) [inline main.cpp]: Don't initialize doubles with integers.
write doubles correctly. 
`4.0` is a double.  `4` is an integer
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp [hunk 3]
```diff
@@ -11,19 +11,20 @@ video: Chapter 2 - Part 3
...
- 
- 
+ This will be the first project where the code you write will be compiled and you will be responsible for making sure it compiles before submitting it for rview.
  
  
+ 1) Write down the names of the 6 major primitive types available in C++ here:
+    int
+    float 
    bool
+    double
```
Comment (matkatmusic) [inline main.cpp]: fix this issue with the `^M`
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
- 10)
- */
+bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
+{
+    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused function
+    return {}; //if your function returns anything other than 'void,' add 'return {}};' to the end of it 
+}
+
+//1)
+bool lowPassFilter(int slopeType = 2, float cutoffFreq = 100.0f)
```
Comment (matkatmusic) [inline main.cpp]: This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```
The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`

- [x] revise

File: main.cpp [hunk 5]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(slopeType, cutoffFreq);
+    return {};
+}
+
+//2)
+int distanceWalkedToday(int numbOfSteps, float milesPerHour = 12.3f)
```
Comment (matkatmusic) [inline main.cpp]: This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```
The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`

- [x] revise

File: main.cpp [hunk 6]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(numbOfSteps, milesPerHour);
+    return {};
+}
+
+//3)
+float initFrequency(bool sawtoothWave, bool squareWave, bool sineWave)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

File: main.cpp [hunk 7]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(sawtoothWave, squareWave, sineWave);
+    return {};
+}
+
+//4)
+double stepLength(int beatDivision, int timeSignature, bool tempoMatching = true)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

File: main.cpp [hunk 8]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(beatDivision, timeSignature, tempoMatching);
+    return {};
+}
+
+//5)
+int busEffects(bool reverbOn = true, bool delayOn = false, int wetDryMix = 50)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

File: main.cpp [hunk 9]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(reverbOn, delayOn, wetDryMix);
+    return {};
+} 
+
+//6)
void defaultLoopLength(int defaultMeasures, int defaultNumOfBeats)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

File: main.cpp [hunk 10]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(defaultMeasures, defaultNumOfBeats);
+}
+
+//7)
+int totalNumOfTracks(int audioTracks, int midiTracks)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

File: main.cpp [hunk 11]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(audioTracks, midiTracks);
+    return {};
+}
+
+//8)
+double equipmentBudget(float percentageOfIncome = 49.9f, int analogSynths = 4, int programmingCourses = 10)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

File: main.cpp [hunk 12]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(percentageOfIncome, analogSynths, programmingCourses);
+    return {};
+}
+
+//9)
+int hamburgersEaten(int withFries = 3, int withoutFries = 2, float withOnions = 0.5f)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

File: main.cpp [hunk 13]
```diff
@@ -56,70 +57,115 @@ video: Chapter 2 - Part 3
...
+{
+    ignoreUnused(withFries, withoutFries, withOnions);
+    return {};
+}
+
+//10)
+void fastFoodReceipts(int mcdonalds = 3, int wendys = 2, int carlsJr = 1)
```
Comment (matkatmusic) [inline main.cpp]: - [x] revise: function name

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Every single function name you chose is a noun.  No verbs were used in the function names, which means they are not describing actions.

File: main.cpp (hunk 4 above)
Comment (luke-den) [reply main.cpp]: Doh!

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**